Puppetfile

```diff
@@ -0,0 +1,8 @@
+## This is an example puppetfile. 
+
+## The commented out section shows the URL of the Puppet forge to use.
```

Comment:
We should not include the "forge" directive. It's more or less deprecated and wrong.